src/lib/staged_ledger/staged_ledger.ml

```diff
@@ -1027,6 +1033,29 @@ module T = struct
       |> Set.to_list
     in
     Ledger.unsafe_preload_accounts_from_parent new_ledger accounts_accessed ;
+    [%log internal] "Check #zkApp does not exceed hardcap" ;
```

Comment:
I think this log is for profiling. It's unlikely that this will significantly impact timings, so it may make sense to remove this log.

Reply:
agree, and otherwise it gets a name like `ZkApp_bound_check`